elide-dbmanager/elide-dbmanager-inmemorydb/src/main/java/com/yahoo/elide/dbmanager/InMemory/InMemoryDB.java

```diff
@@ -48,7 +46,7 @@ public InMemoryDB(Package beanPackage) {
     /**
      * InMemoryDB transaction handler.
      */
-    public static class InMemoryTransaction implements DatabaseTransaction {
+    public class InMemoryTransaction implements DatabaseTransaction {
```

Comment:
Just thinking about this: is there any reason that this class should even be public? I think we can move this class to `private`.